src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,8 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Can we generate the AndroidManifest on demand in the build. It would eliminate hardcoded version numbers.

Reply:
It seems like the file has to exist, but I did figure out how to inject values into it. Injecting the package name was not possible.